Essentials/src/com/earth2me/essentials/UserData.java

```diff
@@ -510,6 +512,34 @@ public void setMuted(boolean set) {
         config.save();
     }
 
+    public String _getMuteReason() {
+        return config.getString("muteReason");
+    }
+
+    public String getMuteReason() {
+        if (muteReason != null) {
+            return muteReason;
+        }
+        else {
```

Comment:
Formatting - `else`s should be on the same line.

Reply:
Also, why not just return `muteReason` even if it *is* null? We have `hasMuteReason` anyway if someone wants to check whether there is a mute reason in advance, and using `null` has an actual symbolic meaning making it clearer what the returned value means.

Reply:
This is a good point, Initially we had it here since we didn't have `hasMuteReason` but since we now have the ability to check, this is obsolete. I will make the change.